daisy_workflows/image_import/inspection/inspection.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Finds boot-related properties of disks.
+
+This module supports offline inspection of block devices and
+virtual disk files, with a focus on information that typically
+requires booting the system.
+
+See `model.InspectionResults` for which information is returned.
+
+In terms of OS support, this module focuses on systems
+that are runnable on GCE, with a particular focus on the
```

Comment:
GCE -> Google Compute Engine, since this is externally visible code.

Reply:
done